actions/rootio/v1/README.md

```diff
@@ -0,0 +1,36 @@
+---
+slug: rootio
+name: rootio
+tags: disk
+maintainers: Dan Finneran <daniel.finneran@gmail.com>
+description: "This action can be used in a variety of ways, however its core functionality
+is the management of disks. The action parses the metadata and will partition, format and
+mount disks"
+version: v1.0.0
+createdAt: "2021-01-20T12:41:45.14Z"
+---
+
+The below example will use the same action in three ways, to "partition" followed by "formating" and
+finally mounting the disks.
+
+```yaml
+    actions:
+      - name: "disk-wipe-partition"
+        image: thebsdbox/rootio:0.0
```

Comment:
Can you change the image with `quay.io/tinkerbell-actions/rootio:v1.0.0`? I will push the image tomorrow

Reply:
The Makefile uses `dockerx` with a `push` should I just build the image locally?

Reply:
I am writing a binary that will use buildkitd to do build across the directories that changed recently. But for testing purposes, you can use them makefile yep. We just need to find a set of the arch we want to support at the beginning and agree on them :D